engine/Engine_Dronecaster.sc

```diff
@@ -31,8 +32,9 @@ Dronecaster {
 
 	start { arg name;
 		if (drones.keys.includes(name), {
+			postln('setting drone: '++name);
 			socket.setSource({
-			    arg hz=440, amp=0.02, amplag=0.02, hzlag=0.01;
+			    arg hz=this.hz, amp=this.amp, amplag=0.02, hzlag=0.01;
```

Comment:
Where do these get initialized now? 
Sorry if it's super obvious and I'm missing it; I haven't been in this engine in a while.

Reply:
oh good point. well in practice they are initialized through the lua params -> commands. but its better if they get an explicit init so adding a commit for that.